cmd/kube-controller-manager/app/core.go

```diff
@@ -485,6 +485,7 @@ func startServiceAccountController(ctx context.Context, controllerContext Contro
 
 func startTTLController(ctx context.Context, controllerContext ControllerContext) (controller.Interface, bool, error) {
```

Comment:
@pohly should we add a name to the logger here?

Reply:
Yes, I think that would be useful. It only needs to be done once per app startup and then makes all log entries generated by the controller more useful.

Doing it here is better than doing it in NewTTLController because during testing of that controller in a unit test we don't need the extra name. That is relevant in the app where we have multiple active controllers.

Reply:
Instead of `ctx`, use something like this:
```
klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "ttl"))
```

Reply:
this is still not resolved. You can check #113622 as an example

Reply:
sorry, I modify it!

Reply:
Please don't bother. I'll clean this up in a separate PR.